MmSupervisorPkg/Private/Guid/MmSupervisorRequestData.h

```diff
@@ -77,12 +89,20 @@ typedef struct _VERSION_INFO_BUFFER {
  **/
 #define   MM_SUPERVISOR_REQUEST_VERSION_INFO  0x0003
 
+/**
+  @retval EFI_INVALID_PARAMETER      If communication buffer is NULL
+  @retval EFI_SECURITY_VIOLATION     If communication buffer is not pointing to designated supervisor buffer
+  @retval EFI_ACCESS_DENIED          If request occurs after MM locked down the request window
+  @retval EFI_COMPROMISED_DATA       Supervisor image buffer does not pass minimal PeCoff check

```

Comment:
Are these return values accurate?

Reply:
Updated a slightly to reflect new status.